x-pack/plugins/actions/server/builtin_action_types/servicenow/schema.ts

```diff
@@ -34,7 +34,7 @@ export const ExecutorSubActionSchema = schema.oneOf([
 ]);
 
 export const ExecutorSubActionPushParamsSchema = schema.object({
-  savedObjectId: schema.string(),
+  savedObjectId: schema.nullable(schema.string()),
```

Comment:
I'm curious if this includes the SO type and spaceId.  We can of course assume the spaceId for now.  Is the type encoded in the string, or assumed to be `alert`?  Could use a comment describing it.

Reply:
I'm not touching the behaviour in this PR, just focusing on decoupling it from Alerting so that it's technically possible to execute.
It *is* in theory assuming this is an Alert, but apparently it isn't actually being used properly yet, so we decided to keep it because we already have this being sent to these services, but once we try to actually use these to generate URLs we'll definitely need to add the type.

Reply:
I'm not touching the behaviour in this PR, just focusing on decoupling it from Alerting so that it's technically possible to execute.
It is in theory assuming this is an Alert, but apparently it isn't actually being used properly yet, so we decided to keep it because we already have this being sent to these services, but once we try to actually use these to generate URLs we'll definitely need to add the type.

Reply:
LOL, github seems to have the diff confused here 🤔